### ISSUE TITLE
Support guile-2.0?

### ISSUE THREAD

**Issue by cebrusfs** [puppeh/vc4-toolchain#11]
Since the debian lastest stable "stretch" source list only contain guile-2.0 instead of guile-1.8. But I got the syntax error when using guile-2.0. Do you have plan to port the binutils to support guile-2.0 syntax?

**Comment by itszor** [comment]
That's something that'd be best done in collaboration with upstream -- I'm not sure it'd be a good idea to attempt it myself. For now you can probably disable guile (automatic file regeneration) support in the build script.

**Comment by cebrusfs** [comment]
I agree they should be done by the upstream. Sorry that I don't know the status of the upstream and I know less about your porting process. Can you please check the upstream is ready for guile-2.0 yet?

**Comment by AntonioND** [comment]
By "disable guile (automatic file regeneration)" do you mean the following lines?
```
# Remove this if you don't want to mess with CGEN.
WITHCGEN="--enable-cgen-maint"
# WITHCGEN=
```

EDIT: Nevermind, I've just seen in the linked issue that it is that option.
